data-raw/healthy-places/noise-complaints.R

```diff
@@ -1,25 +1,51 @@
+# Raw data files were not available to download from online.
+# Noise-complaints data was available from a published reported (sourced below).
+# Data is available in the following file path:
+# healthindexni > data-raw > healthy-places > noise_complaints_data.xlsx
+
+# ---- Load packages ----
 library(tidyverse)
-library(readODS)
 library(httr)
+library(readxl)
+library(geographr)
+
+# ---- Get and clean data ----
+# Noise Complaints Data
+# Source: https://www.daera-ni.gov.uk/sites/default/files/publications/daera/Noise%20Complaint%20Statistics%20for%20NI%202023-24.PDF
+
+noise_complaints_raw <- read_excel("data-raw/healthy-places/noise_complaints_data.xlsx")
```

Comment:
Please can you push these datasets to this branch? They are not included in the path listed here.